contracts/strategy/yearn/YearnStrategy.sol

```diff
@@ -57,6 +59,8 @@ contract YearnStrategy is IStrategy, AccessControl, CustomErrors {
         vault = _vault;
         yVault = IYearnVault(_yVault);
         underlying = IERC20(_underlying);
+
+        underlying.safeIncreaseAllowance(address(this), MAX_UINT256);
```

Comment:
let's change it to "approve" instead

Reply:
for the record, both should do the same, since this is a constructor (unless you use CREATE2, but let's not open that rabbit whole). but approve just feels more correct in this context

Reply:
this fix is already on the main branch, [commit](https://github.com/lindy-labs/sc_solidity-contracts/commit/7aa480c0e96fe632f8c4396df420fec54aa0ca0e)